packages/zosjobs/src/api/DeleteJobs.ts

```diff
@@ -64,7 +64,7 @@ export class DeleteJobs {
         this.log.trace("deleteJobCommon called with parms %s", JSON.stringify(parms));
         ImperativeExpect.keysToBeDefinedAndNonBlank(parms, ["jobname", "jobid"],
             "You must specify jobname and jobid for the job you want to delete.");
-        this.log.info("Deleting job %s(%s). Job modify version?: %s", parms.jobname, parms.jobid, parms.modifyVersion);
+        this.log.info("Deleting job %s (%s). Job modify version?: %s", parms.jobname, parms.jobid, parms.modifyVersion + "");
```

Comment:
I  casted it to string because if the modifyVersion is undefined, the template values in the log string aren't replaced (It comes out as %s). This fixed the log message

Reply:
Thanks for confirming!